### ISSUE TITLE
size-limit doesn't clear bundle folder before each run when --save-bundle option enabled

### ISSUE THREAD

**Issue by MikeDevice** [ai/size-limit#179]
Now there is an interesting behaviour. When I run `size-limit` with `--save-bundle` option, the bundle destination folder doesn't clear before each run. Let's see follow example.

There is a library:
```js
export default () => { /* library logic */ }
export const run = () => { /* run logic */ }
```

I want to calculate the size of run function:

```json
[
  {
    "path": "index.js",
    "import": "{ run }",
    "limit": "100 B"
  }
]
```

```sh
size-limit --save-bundle ./bundle
```

and I get follow files in the `./bundle` directory:
1. `entry.js` with follow content:

```js
import { run } from "<path-from-root>/index.js"
console.log(run)
```

2. `index.js` with whole bundle.

Then I change size-limit config
```diff
[
  {
    "path": "index.js",
-   "import": "{ run }",
    "limit": "100 B"
  }
]
```

and run size-limit again:

```sh
size-limit --save-bundle ./bundle
```
and I get the new `index.js` file and the old `entry.js` file from previous run.

This is wrong behaviour and I think the `--save-bundle` folder should be cleared before each run, but there is another interesting problem. What if someone run `size-limit` like so:

```sh
size-limit --save-bundle ~/photos
```

Then size-limit will removed all the photos and that would be a really bad experience.

What do you think?

**Comment by ai** [comment]
I think we should remove `dir/` from `--save-bundle dir/` call.

Can you send PR?

**Comment by MikeDevice** [comment]
Yes. I'll do.

**Comment by MikeDevice** [comment]
Wait. It seems still implicit to me.

Look. When we call `size-limit --save-bundle dir` the `dir` folder will be removed before each call. And here I think this is implicit behaviour, because user when passes `--save-bundle` flag, is waiting to be saved something, not removed.

I've checked how [clean-webpack-plugin](https://github.com/johnagan/clean-webpack-plugin) handles that. This plugin has `dangerouslyAllowCleanPatternsOutsideProject` option that allows clean patterns outside of `process.cwd()`. By default this plugin doesn't removes anything outside of project folder, and if user wants to override that behaviour, they should pass that flag explicit.

So I think we should make something similar.
If `dir/` is inside of `process.cwd()` -- remove before each call, but if `dir/` is outside -- don't remove until some specific flag like `--force` is not passed.

So what do you think?

**Comment by ai** [comment]
> If dir/ is inside of process.cwd() -- remove before each call, but if dir/ is outside -- don't remove until some specific flag like --force is not passed.

I am not sure that this behavior will be obvious without reading the docs.

> If dir/ is inside of process.cwd() -- remove before each call, but if dir/ is outside -- don't remove until some specific flag like --force is not passed.

We can print an error “the directory is not empty. Pass --clean-dir if you want to remove it”.

**Comment by ai** [comment]
Released in 4.5
